msteams-platform/m365-apps/extend-m365-meeting-app.md

```diff
@@ -0,0 +1,198 @@
+:::image type="content" source="images/teams-meeting-chat-view.png" alt-text="{alt-text}":::
+
+From here, you can skip ahead to [sideload your app with TeamsFx CLI](#sideload-your-app-using-teamsfx-cli) and then [preview your meeting app in Outlook](#preview-your-meeting-app-in-outlook).
+
+## Checklist for enabling meeting apps in Outlook
+
+The requirements for extending meeting apps in Outlook are similar to [extending personal tabs to Outlook](./extend-m365-teams-personal-tab.md), however to sideload your app for testing, you'll need to use TeamsFx CLI (sideloading from Teams is not currently supported).
+
+Extending your Teams meeting app in Outlook involves the following:
+
+> [!div class="checklist"]
+>
+> * [Use app manifest version 1.13 or higher](#use-app-manifest-version-113).
+> * [Use TeamsJS library v2.5.0 or higher](#use-teamsjs-version-250).
+> * [Configure Content Security Policy headers](#configure-csp-headers-for-outlook).
+> * [Update your Microsoft AD App Registration for SSO](#update-azure-ad-app-registration-for-sso).
+> * [Sideload your updated app using TeamsFx CLI](#sideload-your-app-using-teamsfx-cli).
+
+If you have an existing meeting app, make a copy or a branch of your production project for testing and update your App ID in the app manifest to use a new identifier (distinct from the production App ID) for testing.
+
+### Use app manifest version 1.13+
+
+Use app manifest schema version `1.13` or higher for extending Teams apps to run across other Microsoft 365 hosts, including Outlook. If you need to upgrade your app manifest from an earlier version, you have two options:
+
+# [Teams Toolkit](#tab/manifest-teams-toolkit)
+
+1. Open the command palette: `Ctrl+Shift+P`.
+1. Run the `Teams: Upgrade Teams manifest` command and select your app manifest file. Changes are made in place.
+
+# [Manual steps](#tab/manifest-manual)
+
+Open your Teams app manifest and update the `$schema` and `manifestVersion` manually with the appropriate version. Refer to [Teams developer manifest](../resources/schema/manifest-schema.md) for version information:
+
+```json
+{
+    "$schema" : "https://developer.microsoft.com/json-schemas/teams/v1.14/MicrosoftTeams.schema.json",
+    "manifestVersion" : "1.16"
+}
+```
+
+---
+
+### Use TeamsJS version 2.5.0+
+
+Use TeamsJS client library version `2.5.0` or higher to extend your meeting app to run in Outlook. As long as your app uses TeamsJS v2.5.0 or higher, no application code changes are required to run in Outlook.
+
+If you need to upgrade your TeamsJS npm package from v1.x.x to the latest (2.x.x) TeamsJS, you can use Teams Toolkit to help identify and automate the required code changes. Alternately, you can perform the same steps manually; refer to [TeamsJS library](../tabs/how-to/using-teams-client-library.md#whats-new-in-teamsjs-version-2xx) for details.
+
+1. Open the *Command palette*: `Ctrl+Shift+P`.
```

Comment:
I'm not sure I understand the suggestion. The user doesn't need to switch to the Teams Toolkit tab in VS Code because we're accessing the required Toolkit functionality via global VS Code command palette.